influxdb2_operational_monitoring/influxdb2_cardinality_now.yml

```diff
@@ -1,36 +1,60 @@
 apiVersion: influxdata.com/v2alpha1
 kind: Label
 metadata:
-    name: kind-spence-53b001
+    name: determined-torvalds-b69001
 spec:
     color: '#fafafc'
     name: cardinality
 ---
 apiVersion: influxdata.com/v2alpha1
-kind: Variable
+kind: Task
 metadata:
-    name: amazing-boyd-93b003
+    name: unruffled-sinoussi-769001
 spec:
     associations:
       - kind: Label
-        name: kind-spence-53b001
-    language: flux
-    name: bucket
+        name: determined-torvalds-b69001
+    every: 1d
```

Comment:
1d might not be often enough.  Maybe adjust this, and also line 41 to use a 1h interval?